### PR TITLE
Add pieces and sectors diagrams

### PR DESCRIPTION
**Pull request by mishmosh**
- Add pieces diagram and intro text to 2.2.2 Piece.
- Add sectors diagram and intro text to 2.6.2 Sector.

### CODE REVIEW

**Comment by mishmosh** [comment]
Sealed/unsealed sectors diagram to come. I think it belongs in a new subsection (either before or after Data Transfer), rather than under Pieces, but alt suggestions are welcome.

**Comment by mishmosh** [comment]
Ready for @sternhenri review (and, if no issues, merge). Please check rendering.

@hannahhoward @vmx Thanks for the nudge to add this info. Your input on this PR is very welcome, but not required if you are busy.

File: src/systems/filecoin_mining/sector/_index.md [hunk 1]
```diff
@@ -11,12 +11,18 @@ entries:
 The `Sector` is a fundamental "storage container" abstraction used in Filecoin Storage Mining. It is the basic unit of storage,
 and serves to make storage conform to a set of expectations.
 
+New sectors are empty upon creation. As the miner receives client data, they fill or "pack" the piece(s) into an unsealed sector.
+
+Once a sector is full, the unsealed sector is combined by a proving tree into a single root `UnsealedSectorCID`. The sealing process then encodes an unsealed sector into a sealed sector, with the root `SealedSectorID`.
```
Comment (vmx) [inline src/systems/filecoin_mining/sector/_index.md]: Typo: `SealedSectorID` -> `SealedSector*C*ID`.

Comment (sternhenri) [reply src/systems/filecoin_mining/sector/_index.md]: Thanks VMX, the CBOR encoding is mentioned elsewhere in the spec as the encoding for all objects.

Comment (vmx) [reply src/systems/filecoin_mining/sector/_index.md]: @sternhenri In the published version https://filecoin-project.github.io/specs/ CBOR is mentioned here and there in sub-sections, but there is no general note that things are generally encoded as CBOR and hence also have the multicodec codec for DagCBOR.

**Pull request merged**